### PR TITLE
go/mio: fix build after arm64 port

### PR DESCRIPTION
**Pull request by andriytk**
Fix build after commit bc844a8 (ARM64 platform support) by including config.h file.

### CODE REVIEW

File: bindings/go/mio/mio.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ package mio
 // #cgo CFLAGS: -Wno-attributes
 // #cgo LDFLAGS: -L../../../motr/.libs -Wl,-rpath=../../../motr/.libs -lmotr
 // #include <stdlib.h>
+// #include "config.h"
```
Comment (huanghua78) [inline bindings/go/mio/mio.go:48]: Can you take a look at this file:
https://github.com/Seagate/cortx-motr/blob/main/motr/examples/example1.c
```
/*
 * Please change the dir according to you development environment.
 *
 * How to build:
 * gcc -I/work/cortx-motr -I/work/cortx-motr/extra-libs/galois/include \
 *     -DM0_EXTERN=extern -DM0_INTERNAL= -Wno-attributes               \
 *     -L/work/cortx-motr/motr/.libs -lmotr                            \
 *     example1.c -o example1
```

Maybe it also needs to be updated?
We can do it in another PR.

**Review by siningwuseagate — APPROVED** [review]
